README.md

```diff
@@ -18,9 +18,11 @@ You'll need at least one emulator.  I've found the easiest way to add Virtual De
   *  Click 'Create Virtual Device' - bottom left-hand corner
   *  Create at least one using the latest Android Release (Nougat).  I have a few different size and Android Release configurations
 
-Also you'll need a Gradle properties file outside the project for keeping secrets that aren't checked into source control.  This should be in ~/.gradle/gradle.properties and contain the following:
-`MYAPP_RELEASE_STORE_FILE=
-MYAPP_RELEASE_KEY_ALIAS=`
+Also you'll need a Gradle properties file outside the project for keeping secrets that aren't checked into source control. This should be in ~/.gradle/gradle.properties and contain the following:
+`MYAPP_RELEASE_STORE_FILE=`    
+`MYAPP_RELEASE_KEY_ALIAS=`
+
+Which should contain the keystore file name and alias. Talk to either Will Granger (will@zooniverse.org) or Noah Malmed (noah@zooniverse.org) for more information on this.
```

Comment:
We should think of other places to keep this keystore file instead of it being dependent on us.

Reply:
There is a way to have google manage this for you, I'll write up an issue

Reply:
https://github.com/zooniverse/mobile/issues/205